engines/saga/itedata.cpp

```diff
@@ -482,9 +504,15 @@ const char *pieceNames[][PUZZLE_PIECES] = {
 		"cacciavite", "pinze", "morsa", "morsa da legno", "livella",
 		"spago", "pialla", "martello", "metro a nastro", "accetta",
 		"cesoie", "righello", "sega", "mazza", "pennello"
+	},
+	{
+		"tournevis", "pinces", "\202taux \205 vis", "\202taux en bois", "niveau",
```

Comment:
`pinces` and `étaux` are plural, the other languages seem to use singular.

Reply:
Yes it is an error from the original translation i don't see :(
It is better singular.

Reply:
"tournevis", "pince", "\202tau \205 vis", "pince \205 bois", "niveau",